interface-definitions/vrrp.xml.in

```diff
@@ -222,6 +222,21 @@
                   </valueHelp>
                 </properties>
               </leafNode>
+              <leafNode name="virtual-address-excluded">
+                <properties>
+                  <multi/>
+                  <help>Virtual address (If you need additional IPv4 and IPv6 in same group)</help>
+                  <constraint>
+                    <validator name="ipv4-host"/>
+                    <validator name="ipv6-host"/>
+                  </constraint>
+                  <constraintErrorMessage>Virtual address must be a valid IPv4 or IPv6 address with prefix length (e.g. 192.0.2.3/24 or 2001:db8:ff::10/64)</constraintErrorMessage>
+                  <valueHelp>
+                    <format>&lt;IPv4|IPv6&gt;</format>
```

Comment:
Please specify individual `<valueHelp>` options for IPv4 and IPv6 addresses. See e.g. SSH which use ipv4-address, too.

Reply:
@c-po done.